fedora/templates/csv/services_enabled.csv

```diff
@@ -3,5 +3,6 @@ chronyd,chrony,
 crond,cronie,
 firewalld,firewalld,
 ntpd,ntp,
+pcscd,pcsc-lite,
```

Comment:
This is related to rule `linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_auth/`. Your PR adds this service to the list of enabled services for Fedora, but the OVAL for that rule is not enabled on Fedora, and the remediation scripts are not enabled on Fedora as well. I think you need to add multi_platform_fedora there.